content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -1,15 +1,16 @@
 # DVC Experiments Overview
 
-DVC Experiments are captured automatically by DVC when [run]. Each experiment
-creates and tracks a variation of your data science project based on the changes
-in your <abbr>workspace</abbr>.
+DVC Experiments are captured automatically by DVC when [run], but can also be
+[saved] manually. Each experiment creates and tracks a variation of your data
```

Comment:
can be captured automatically when [run with DVC] or [saved] manually (e.g. after running your code manually).